src/Sulu/Component/Content/Metadata/ItemMetadata.php

```diff
@@ -245,20 +245,33 @@ public function getTitle($locale)
     }
 
     /**
-     * Return the paramter with the given name.
+     * Return the parameter with the given name.
      *
      * @param string $name
      */
-    public function getParameter($name)
+    public function getParameter($name, bool $nullIfNotExists = false)
```

Comment:
Not a big fan of these kind of flags... I have tried to see if there are any usages of this function, and there don't seem to be any... Can't we just return null if nothing exists without a flag then?

Reply:
Should I also refactor this whole method to ignore all keys and just use the `['name' => 'foo']` property?

Reply:
I think that's related to my question in https://github.com/sulu/sulu/pull/5666#discussion_r534032459. If I understand you correctly, this function was not working at all before? Resp. are there sometimes different representations, and the current implementation works for both of them?

Reply:
Yes, that implementation did work for both representations, but I think it's not necessary, since `getParameter()` was not used somewhere else ...